dashboard/app/views/levels/editors/_all.html.haml

```diff
@@ -47,15 +47,20 @@
     updateVideoPreview();
   });
 
+:css
+  .required:after {
+    content: ' *';
+    color: #ff0000;
+  }
 .field.callouteditor
   %div{id: 'callout_template', style: 'display: none;'}
     %div{class: 'callout_space'}
       %label Callout
       %table
         %tr
-          %td Element ID
-          %td Localization Key
-          %td Callout text
+          %td{class: 'required'} Element ID
```

Comment:
(and incidentally you might notice from the above line `.field.callouteditor` that you can omit the %div if you have a .class